it/v1/endpoints/AmendOtherReliefsControllerISpec.scala

```diff
@@ -305,7 +322,7 @@ class AmendOtherReliefsControllerISpec extends IntegrationBaseSpec {
           |  ]
           |}""".stripMargin)
 
-      val allInvalidvalueFormatRequestBodyJson = Json.parse("""
+      val allInvalidValueFormatRequestBodyJson = Json.parse("""
```

Comment:
I think this whole test spec would be easer to read if all the JSONs were moved into a companion object (object AmendOtherReliefsControllerISpec) so that they are out of the way

Reply:
also can we move the test trait to the bottom of the spec again so it is out of the way and the tests are the first thing we see when opening this file